gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/BooleanValue.java

```diff
@@ -57,6 +57,32 @@ private Builder() {
     }
 
     @Override
+    public Boolean get() {
```

Comment:
Here and elsewhere (possibly not related to this PR) do you think that getters should be in the builder? I think that elsewhere in `gcloud-java` we don't have them.

This reasoning also applies to other getters: `getMeaning`, `getIndexed`, `getValueType`.